rdkit/Chem/fmcs/fmcs.py

```diff
@@ -827,6 +815,9 @@ def get_typed_fragment(typed_mol, atom_indices):
       orig_bonds.append(orig_bond)
     elif count == 1:
       raise AssertionError("connected/disconnected atoms?")
+    elif count == 0:
+      # This means that the bond is not in the fragment or the source code above is the error.
+      raise AssertionError("Error Source Code?")
```

Comment:
Why add an exception if the original code doesn't contain one.

Reply:
I am so sorry, this is my mistake. I will revert it to the original